safe-global/safe-wallet-web
# chore: update deployment-related packages

## What it solves

Resolves outdated deployment support

## How this PR fixes it

All deployment-related packages have been updated:

- `@safe-global/safe-deployments`
- `@safe-global/safe-core-sdk`
- `@safe-global/safe-core-sdk-utils`
- `@safe-global/safe-ethers-lib`

## How to test it

Open Safes across networks and observe no issue regarding SDK instantiation (there should be no error when loading the Safe). Creating and executing transactions should also work as is.

## Checklist
